dask/multiprocessing.py

```diff
@@ -171,13 +172,16 @@ def get(
         If True [default], `fuse` is applied to the graph before computation.
     pool : Executor or Pool
         Some sort of `Executor` or `Pool` to use
+    initializer: function
+        Function to initialize a worker process before running any tasks in it.
     chunksize: int, optional
         Size of chunks to use when dispatching work.
         Defaults to 5 as some batching is helpful.
         If -1, will be computed to evenly divide ready work across workers.
     """
     chunksize = chunksize or config.get("chunksize", 6)
     pool = pool or config.get("pool", None)
+    initializer = initializer or config.get("initializer", None)
```

Comment:
This config name seems a bit vague. Perhaps we should clarify it a bit

Also depending on how we handle calling the intializer, it could make sense to handle the default here too

```suggestion
    initializer = initializer or config.get("multiprocessing.initializer", None) or initialize_worker_process
```